src/Evolution/Systems/RadiationTransport/M1Grey/Initialize.hpp

```diff
@@ -9,6 +9,8 @@
 #include <utility>  // IWYU pragma: keep  // for move
```

Comment:
What is the reason for the changes in this commit?  We don't want to do this if we can avoid it.

Reply:
In `M1HydroCoupling.hpp` the `return_tags` for the `ComputeM1HydroCoupling` struct and `ComputeM1HydroCouplingJacobian` struct need these tags.

Reply:
Those are outputs, though.  Why do you need to compute them?

Reply:
We will compute them in the EvolveSystem (below link).  My understanding is, even if they're return_tags, they need to be in the Databox?  In `Initialize.hpp`, I see 
```
using m1_variables_tag = typename System::primitive_variables_tag;
```
Do the tags need to be prepended with `::Source` and `::jacobian` in System.hpp and then added to `primitive_variables_tag`?

https://github.com/sxs-collaboration/spectre/blob/887e81c4a148d04634b913185a2ecd368d142152/src/Evolution/Executables/RadiationTransport/M1Grey/EvolveM1Grey.hpp#L242

Reply:
But *why* are you computing them?  What uses them?

Let me make an analogy with runtime code:
```c++
const auto energy = ...;
Scalar<DataVector> energy_source{};
compute_source(make_not_null(&energy_source), energy);
return energy;
```
I am asking the equivalent of "Why is `energy_source` needed?" and getting the answer "It's a required argument of the the `compute_source` function.".  As far as I can tell, you write to these variables, but never read from them, which doesn't seem useful.

Reply:
Following the SpECTRE discussion, I understand your question now.  The short answer is we do not need these changes, yet.  Formally, this PR does not couple the neutrinos to the fluid.  So nothing, in this PR, needs those terms.  In the future, they will be needed.   

The reason why I included these changes is I want to see if the changes on the original M1 branch were able to compile.  Specifically, in the EvolveM1Grey file, the initialization and step action calls to HydroCoupling were commented out. 

https://github.com/wthrowe/spectre/blob/b73281b3c6c27e029d55097d4fec1f6144bd04a3/src/Evolution/Executables/RadiationTransport/M1Grey/EvolveM1Grey.hpp#L433

After including them in the Evolve system, I found it could not compile, which led me to discover the tags are needed in the databox.  For this PR, I will not include these changes.  For posterity, (myself mostly) the tags for Jacobians of the stiff hydro coupling sources will need to be included.

Reply:
They were commented out on the branch because they were left over from the old non-IMEX version.  They were supposed to be deleted when I cleaned up the branch, not uncommented.

You do not need the sources and jacobians in the DataBox in order to couple to fluids.  You can calculate them during the computation of the hydro sources without permanently storing them.